packages/core/utils/src/dal/mikro-orm/mikro-orm-repository.ts

```diff
@@ -622,18 +661,38 @@ export function mikroOrmBaseRepositoryFactory<T extends object = object>(
             })
           })
 
-          await this.upsertMany_(manager, relation.type, normalizedData)
+          const { performedActions: performedActions_ } =
+            await this.upsertMany_(manager, relation.type, normalizedData)
+          this.mergePerformedActions(performedActions, performedActions_)
         }
 
+        const toDeleteEntities = await manager.find<any>(
```

Comment:
I assume this should work as the entityMap will be empty, so it will go straight to the DB, right?

Reply:
Ues that's right 👍